src/api/project/repository/project_repository.go

```diff
@@ -20,8 +20,32 @@ func NewProjectRepository(db *gorm.DB) domain.Repository {
 	}
 }
 
+func (t *ProjectRepository) GetAll(_ context.Context) ([]*model.Projects, error) {
+	var projects []*model.Projects
+
+	err := t.DB.Order("id desc").Find(&projects).Error
+	if err != nil {
+		log.Printf("Error getting Tasks %v", err)
+		return nil, err
+	}
+
+	return projects, nil
+}
+
+func (t *ProjectRepository) GetById(_ context.Context, id string) (*model.Projects, error) {
+	var project *model.Projects
+
+	err := t.DB.First(&project, id).Error
+	if err != nil {
+		log.Printf("Error getting Task %v", err)
```

Comment:
Am I understanding this wrong or the log output should be "Error getting Project %v"?